pylint/checkers/exceptions.py

```diff
@@ -254,6 +253,18 @@ class ExceptionsChecker(checkers.BaseChecker):
 
     def open(self) -> None:
         self._builtin_exceptions = _builtin_exceptions()
+        for exc_name in self.linter.config.overgeneral_exceptions:
+            if "." not in exc_name:
+                warnings.warn_explicit(
+                    "Specifying exception names in overgeneral-exceptions option"
+                    " without module name is deprecated and support for it"
+                    " will be removed in pylint 3.0."
+                    f" Use fully qualified name (e.g. builtins.{exc_name}) instead.",
```

Comment:
```suggestion
                    f" Use fully qualified name (e.g. 'builtins.Exception' or 'mypackage.MyCustomException') instead.",
```
Maybe it's clearer as an example ?

Reply:
This was suggested here:
https://github.com/PyCQA/pylint/pull/7497#discussion_r975003974

Reply:
For now, I added quotes to the exception name at least.